README.md

```diff
@@ -20,6 +20,15 @@ You can use the framework in various services such as social networks, call cent
 * The framework is being actively maintained and thoroughly tested. The team is open to suggestions and quickly reacts to bug reports.
 
 # Quick Start
+
+## System Requirements
+
+- Supported operating systems include Ubuntu 18.04+, Windows 10+ (partial support), and MacOS Big Sur (partial support);
+- Python version 3.8 or higher is necessary for proper functionality;
+- A minimum of 1 GB of RAM is required for optimal performance;
```

Comment:
These words don't mix well together. ("minimum", "required" and "optimal" are all about putting some restrictions on the amount of RAM)
```suggestion
- 1 GB of RAM is recommended for optimal performance;
```